tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+import shutil
+
+import pytest
+
+from ib_manifest_util import TEST_DATA_DIR
+
+# @pytest.fixture
+# def repo_dir():
+#     """Pathlib Path to the sample IB repo folder"""
+#     print(f'test data dir: {TEST_DATA_DIR}')
+#     return TEST_DATA_DIR.joinpath("sample_repo")
+
```

Comment:
```suggestion
```

Can we remove this?

Reply:
sorry, I should've double-checked for commented code!